paragrafer/strukturLedelseOgOrganisasjon.tex

```diff
@@ -130,14 +130,14 @@ \subsection{Informatikernes IT-ekskursjon}{
 Komiteens hovedoppgave er å arrangere ekskursjon til Oslo for masterstudenter hver høst. Komiteens navn forkortes Itex.
 }
 
-\subsection{Applikasjonskomiteen}{
-Komiteens hovedoppgave er å utvikle og drifte egne it-systemer. Komiteens navn forkortes appkom.
-}
-
 \subsection{Casual gaming}{
 Komiteens hovedoppgave er å organisere LAN. Casual Gaming opererer frittstående fra linjeforeningen.
 }
 
+\subsection{Realfagskjelleren}{
+Realfagskjelleren Realfagskjellerens hovedoppgave er å opprettholde et sosialt lavterskeltilbud for studentene ved Volvox & Alkymisten, Delta, Spanskrøret og Online. Realfagskjelleren er frittstående fra linjeforeningen.	
```

Comment:
Man skal bruke fullstendige navn om igjen i lovtekster, selv om det virker repetitivt. Dette er ikke noe som er ment for lesing på sengekanten ;-)